src/Alpha_complex/test/Weighted_alpha_complex_unit_test.cpp

```diff
@@ -179,8 +179,8 @@ BOOST_AUTO_TEST_CASE(Weighted_alpha_complex_3d_comparison) {
       std::clog << std::endl;
       BOOST_CHECK(false);
     }
-    // I had to make a hard limit as it is converted from Kernel::FT
-    if (std::fabs(d3_itr->second - dD_itr->second) > 1e-5) {
+    // In safe mode, relative error is less than 1e-5 (can be changed with set_relative_precision_of_to_double)
+    if (std::fabs(d3_itr->second - dD_itr->second) / std::fabs(d3_itr->second) > 1e-5) {
```

Comment:
In theory, they could both have a relative error (with respect to the exact value) of 1e-5 in opposite directions, making the relative difference closer to 2e-5.
```c++
std::fabs(d3_itr->second - dD_itr->second) > 1e-5 * (std::fabs(d3_itr->second) + std::fabs(dD_itr->second))
```
may help with that, and also avoids the subtlety of relying on the fact that 0/0 is NaN which is not > 1e-5.

Reply:
I used your code on 05e28e39